sdk/src/marginCalculation.ts

```diff
@@ -0,0 +1,351 @@
+import { BN } from './';
```

Comment:
might move this to margin file

Reply:
oh I see you are saying just keep this in the margin.ts file don't make a separate marginCalculation.ts file?